packages/keycloak/test/decorator/http.test.ts

```diff
@@ -0,0 +1,98 @@
+import { Test } from '@nestjs/testing'
+import { params, suite, test } from '@testdeck/jest'
+import { BaseTest } from '../base-test'
+import { GqlModule } from './gql.module'
+import { RestModule } from './rest.module'
+
+@suite('HTTP')
```

Comment:
```suggestion
@suite('Protected Decorator')
```

I think we can put the `decorator name` to this file :thinking:

Reply:
Not sure. We'll have other decorators that will be tested here!